docs/maintenance/Issues.mdx

```diff
@@ -66,16 +75,30 @@ If your link is both a "permalink" (includes a commit hash instead of a branch n
 When viewing a file in GitHub pressing `y` will update the URL to a "permalink" with the current commit hash, then you can select the relevant lines and paste that URL into the comment.
 :::
 
+### Looking for Duplicates
+
+It's worth noting that, occasionally, a user will intentionally raise a duplicate issue because they feel the original issue was closed when it shouldn't have been.
+If this is the case, you should read the original issue to gather context, understand the reason for it being closed, and determine if the new issue raises any new or relevant point that requires addressing.
+
 ### Determining Whether Code is Working As Intended
 
 As you become more familiar with the codebase and how everything works, this will be easier to do intuitively, but to begin with, this will likely involve investigating the documentation, code, and tests to determine if it's a bug or working as intended.
 In general, if there is a passing test or documented example that is the same as or similar to the repro code — that indicates it's working as intended.
 If you can't find anything that matches, use your best judgement based on the spirit of the code.
 
-### Looking for Duplicates
+### Community Engagement Evaluation
 
-It's worth noting that, occasionally, a user will intentionally raise a duplicate issue because they feel the original issue was closed when it shouldn't have been.
-If this is the case, you should read the original issue to gather context, understand the reason for it being closed, and determine if the new issue raises any new or relevant point that requires addressing.
+In most cases, maintainers have a pretty good idea of how people write code and what most users of the typescript-eslint tooling want.
+However, there are cases when maintainers can't confidently choose the most reasonable approach to solving a particular problem:
+
+- The issue subject relates to a library/framework that the maintainers are not familiar with. Therefore, they don't know the idiomatic patterns associated with it.
+- There is a new syntax or a new feature - sometimes it's hard to guess how people might use that feature.
+- The issue was about to be closed, but someone made a compelling argument. This requires further discussion to find a viewpoint that most people agree on.
+
```

Comment:
This way folks know we're giving them tools to try things out & build community consensus:

```suggestion

For requests that can be implemented outside of typescript-eslint, be sure to mention any relevant APIs such as [Custom Rules](../developers/Custom_Rules.mdx) that can be used.

```